tests/test_utils.py

```diff
@@ -888,7 +914,7 @@ def test_sync_core_json_files(
         actual_log_messages = [
             c[1]["message"] for c in mock_log_message.call_args_list
         ]
-        self.assertEqual(expected_logs, actual_log_messages)
+        self.assertCountEqual(expected_logs, actual_log_messages)
```

Comment:
This was updated because the order of results from `aind_data_schema.utils.json_writer.SchemaWriter.get_schemas()` was inconsistent, causing previous tests to fail.